pallets/root-of-trust/src/benchmarking.rs

```diff
@@ -22,11 +22,17 @@
 
 use super::*;
 
-use frame_benchmarking::{account, benchmarks};
+use frame_benchmarking::{
+	benchmarks,
+	account,
+	impl_benchmark_test_suite,
+};
 use frame_system::RawOrigin;
 use sp_runtime::traits::Bounded;
 use sp_std::{prelude::*, vec};
 
+use crate::Pallet as Rot;
```

Comment:
Let's swap `Rot` to `RootOfTrust` for clarity purposes

Reply:
Sounds good, will update it :+1: